bicep/main.bicep

```diff
@@ -79,6 +79,10 @@ param sendGridEmailFrom string
 @description('The name of the Send Grid Email From Name.')
 param sendGridEmailFromName string
 
+// When set to 'true', Parameters 'sendGridKeySecretValue' and 'sendGridEmailFrom' should be provided
+@description('Set if integration with SendGrid is enabled.')
+param sendGridIntegrationEnabled string
```

Comment:
Would it make sense that instead of setting this new variable we just create condition on if sendgrip api key value is set or not and default value to empty?

That way we can just say if you have sendgrid api key, add it